<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page1</class>
 <widget class="QWidget" name="page1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_led">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开灯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_beer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开蜂鸣器</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_light">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
